source/index.rst

```diff
@@ -12,12 +12,18 @@ Data formats for gamma-ray astronomy
 .. image:: https://licensebuttons.net/l/by/4.0/88x31.png
     :target: https://creativecommons.org/licenses/by/4.0/
 
-A place to propose and share data format descriptions for gamma-ray astronomy.
+The *Data formats for gamma-ray astronomy* is a community-driven initiative for the definition of a common and open high-level data format for gamma-ray instruments.
 
 * Repository: https://github.com/open-gamma-ray-astro/gamma-astro-data-formats
 * Docs: https://gamma-astro-data-formats.readthedocs.io/
 * Mailing list: https://lists.nasa.gov/mailman/listinfo/open-gamma-ray-astro
 
+References
+==========
+
+The following paper describes the context of this initiative and its evolution:
```

Comment:
I think you need a newline here to make it actually render as a bullet point list in rst. 

You can try to build locally using the instructions in the readme.

Reply:
added